src/Features/Core/Portable/InlineMethod/AbstractInlineMethodRefactoringProvider.InlineContext.cs

```diff
@@ -400,17 +400,19 @@ private LocalVariableDeclarationVisitor(CancellationToken cancellationToken)
             {
                 var visitor = new LocalVariableDeclarationVisitor(cancellationToken);
                 var operation = semanticModel.GetOperation(methodDeclarationSyntax, cancellationToken);
-                if (operation != null)
-                {
-                    visitor.Visit(operation);
-                }
+                visitor.Visit(operation);
 
                 return visitor._allSymbols.ToImmutableHashSet();
             }
 
-            public override void Visit(IOperation operation)
+            public override void Visit(IOperation? operation)
```

Comment:
📝 This should override `DefaultVisit` instead of `Visit`, at which point neither the nullable argument nor the null check are necessary

Reply:
I'm not going to change the implementation of a visitor in this PR. If you want to do a followup to change that, it's fine with me.